face_model/process_image.py

```
@@ -1,33 +1,29 @@
-import numpy as np
 import os
-import csv
-from PIL import Image
-from ultralytics import YOLO
-from argface_model.argface_classifier import ArcFaceClassifier
 import uuid
+import logging
```

Ruff

`logging` imported but unused

Remove unused import: `logging`

(F401)